lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -1558,18 +1562,21 @@ bool SessionState::is_active() {
 }
 
 void SessionState::set_fsm_state(
-    SessionFsmState new_state, SessionStateUpdateCriteria* session_uc) {
+    SessionFsmState new_state, SessionStateUpdateCriteria* uc) {
   // Only log and reflect change into update criteria if the state is new
+  uint32_t local_teid_ = get_upf_local_teid();
   if (curr_state_ != new_state) {
-    MLOG(MDEBUG) << "Session " << session_id_ << " Teid " << local_teid_
+    MLOG(MDEBUG) << "Session id: " << session_id_ << " Teid: " << local_teid_
                  << " FSM state change from "
                  << session_fsm_state_to_str(curr_state_) << " to "
-                 << session_fsm_state_to_str(new_state);
-    curr_state_ = new_state;
-    if (session_uc) {
```

Comment:
this nullptr check is important, why get rid of it?

Reply:
Re added the nullptr check as suggested.